docs/README.md

```diff
@@ -423,6 +423,35 @@ Renders a filter selector for the fetched results.
 | `updateOnFilterSelectionOnMobile` | `boolean` | Whether the search results on mobile should be updated according to filter selection (`true`) or not (`false`). Notice: this prop only works if the `preventRouteChange` prop is declared as `true`.                                                                        | `false`       |
 | `showClearByFilter`       | `boolean` | Whether a clear button (responsible for erasing all filter options selected by the user) should be displayed alongside the filter name (`true`) or not (`false`).                                                                                                                              | `false`       |
 | `priceRangeLayout` | `enum` | Whether a text field to enter the desired price range should be displayed  (`inputAndSlider`) or not (`slider`). | `slider` |
+| `facetOrdering` | `[object]` | Apply custom sorting rules for filters. The default behavior sorts descending the items by quantity. | `undefined` |
```

Comment:
```suggestion
| `facetOrdering` | `object` | Apply custom sorting rules for filters. The default behavior sorts descending the items by quantity. | `undefined` |
```

Reply:
I agree! Just pushed a new version following what I've found in other VTEX projects. LMK your thoughts. Thanks!